test/Interop/Cxx/function/emit-called-function-irgen.swift

```diff
@@ -0,0 +1,20 @@
+// Test that Swift emits code for a C++ inline function that is called from
```

Comment:
Please add "inline" to the names of the tests.

Reply:
Done.